pulseapi/entries/views.py

```diff
@@ -525,10 +525,9 @@ def get_serializer_context(self):
     def post(self, request, *args, **kwargs):
         request_data = request.data
         user = request.user if hasattr(request, 'user') else None
-
         validation_result = post_validate(request)
 
-        if validation_result:
+        if validation_result is True:
```

Comment:
@cadecairos so... yeah...  hmm.

Reply:
/facepalm

Reply:
"When JavaScript Developers start coding in Python"

Reply:
Oh, no, this would **absolutely** be the same bug in JS, because `validation_result` can be `True` _or string data_

Reply:
In fact, we had the correct check _in the same file, for the same var_ so I don't even know how this got here.